### ISSUE TITLE
Scale won't work on HTML?

### ISSUE THREAD

**Issue by ghost** [scratchblocks/scratchblocks#393]
(I'm not good at English and JavaScript.)
I'm trying to use this on my website, but scale seems doesn't work.
``` html
<!DOCTYPE html>
<html>
  <head>
    <script src="/scratch_blocks.js"></script>
  </head>
  <body>
    <p class="blocks">when green flag clicked</p>
    <span class="blocks">when green flag clicked</span>
    <script>scratchblocks.renderMatching("p.blocks", {style: "scratch3", scale: 1});scratchblocks.renderMatching("span.blocks", {style: "scratch3", scale: 1, inline: true})</script>
  </body>
</html>
```
out:
![out](https://i.imgur.com/Bb5Lxfc.png)
``` html
<!DOCTYPE html>
<html>
  <head>
    <script src="/scratch_blocks.js"></script>
  </head>
  <body>
    <p class="blocks">when green flag clicked</p>
    <span class="blocks">when green flag clicked</span>
    <script>scratchblocks.renderMatching("p.blocks", {style: "scratch3", scale: 0.675,});scratchblocks.renderMatching("span.blocks", {style: "scratch3", scale: 0.675, inline: true})</script>
  </body>
</html>
```
out still is:
![out2](https://i.imgur.com/CRX5hkJ.png)
It haven't changed, why?
How can I fix it?

**Comment by tjvr** [comment]
Which version of scratchblocks are you using?

**Comment by o0Rainbow0o** [comment]
> Which version of scratchblocks are you using?

v3.5.2, latest version.

**Comment by apple502j** [comment]
`scale` is not in 3.5.2.

**Comment by o0Rainbow0o** [comment]
> `scale` is not in 3.5.2.

So... README.md is too new?
Then how I change the scale in another way?  It's really too large.

**Comment by CST1229** [comment]
> So... README.md is too new?

It's not, the README is for the current state of the repo, which is not the current release.

>  Then how I change the scale in another way? It's really too large.

You can try building scratchblocks yourself and using that, maybe.

**Comment by tjvr** [comment]
I can upload a new version! I might wait until #389 is merged 👍

**Comment by CST1229** [comment]
> I might wait until #389 is merged 👍

Speaking of merging PRs, what about #390?

**Comment by o0Rainbow0o** [comment]
>It's not, the README is for the current state of the repo, which is not the current release. Essentially it's like a beta.

I see. Thank you.

**Comment by themysticsavages** [comment]
**I found a temporary solution!**
Once you render the scratchblock element, a `div` element is created with the SVG for the blocks inside. You can simply edit the width and height of the SVG like so:
```js
let svgw = document.getElementsByTagName('svg').item(0).width.baseVal.value,
    svgh = document.getElementsByTagName('svg').item(0).height.baseVal.value

document.getElementsByTagName('svg').item(0).width.baseVal.value = svgw / 1.5
document.getElementsByTagName('svg').item(0).height.baseVal.value = svgh / 1.5
```

**Comment by tjvr** [comment]
I've just published 3.6.0! See #403.
